---
int8 playerNum
int8 teamColor    
int8 fallen
float32[3] pose
float32[2] walkingTo
float32[2] shootingTo
int32 ballAge    
float32[2] ball
float32[2] ballVel
int16 intention
int16 numOfDataBytes
